tests/test_check_deleted_symbols.py

```
@@ -265,7 +265,7 @@ def test_deleted_class_added_after_merge_base_fails(self, tmp_path: Path):
         _checkout(repo, "main")
         _git(repo, "merge", "pr-branch", "--no-edit")
 
-        violations, waived = cds.check_deleted_symbols(base_tip, repo)
+        violations, waived, _ = cds.check_deleted_symbols(base_tip, repo)
```

Ruff (0.16.1)

[warning] Unpacked variable `waived` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)